db/seeds.rb

```diff
@@ -14,3 +14,21 @@
   ReasonType.create(description: criteria)
 end
 
+unless Rails.env.production?
+  agency = Agency.create
+  user = User.new
+  user.email = "admin@hifinf.org"
```

Comment:
We're not using protected_attributes, so all the attributes could be passed to User.new as a hash